README.md

```diff
@@ -50,6 +51,35 @@ The 9 NWP variables, from Open-Meteo documentation, are mentioned above with the
 - The model is trained on [UK MetOffice](https://www.metoffice.gov.uk/services/data/met-office-weather-datahub) NWPs, but when running inference we use [GFS](https://www.ncei.noaa.gov/products/weather-climate-models/global-forecast) data from [Open-meteo](https://open-meteo.com/). The differences between GFS and UK MetOffice, could led to some odd behaviours.
 - It looks like the GFS data on Open-Meteo is only available for free for the last 3 months. 
 
+## Evaluation
+
+To evaluate the model we use the [UK PV](https://huggingface.co/datasets/openclimatefix/uk_pv) dataset and the [ICON NWP](https://huggingface.co/datasets/openclimatefix/dwd-icon-eu) dataset.
+All the data is publicly available and the evaluation script can be run with the following command
+
+```bash
+python scripts/run_evaluation.py
+```
+
+The test dataset we used is defined in `quartz_solar_forecast/dataset/testset.csv`. 
+This contains 50 PV sites, which 50 unique timestamps. The data is from 2021. 
+
+The results of the evaluation are as follows
+The MAE is 0.1906 kw across all horizons. 
+
+| Horizons | MAE [kw]      | MAE [%] |
+|----------|---------------| ------- |
+| 0        | 0.202 +- 0.03 | 6.2 |
+| 1        | 0.211 +- 0.03 | 6.4 |
+| 2        | 0.216 +- 0.03 | 6.5 |
+| 3 - 4    | 0.211 +- 0.02 |6.3 |
```

Comment:
It's very odd the 0th hour horizon is the same as the 24-48 hour horizon.

Reply:
Ive added this to the Readme.md under the evaluation

Reply:
Okay, that could explain it. Is the current model trained without any pv_dropout at all?